services/device-registry-jdbc/src/test/java/org/eclipse/hono/deviceregistry/jdbc/impl/AbstractJdbcRegistryTest.java

```diff
@@ -76,8 +76,8 @@ enum DatabaseType {
     }
     protected static final Span SPAN = NoopSpan.INSTANCE;
 
-    private static final DatabaseType DEFAULT_DATABASE_TYPE = DatabaseType.H2;
-    private static final DatabaseType DATABASE_TYPE = DatabaseType.valueOf(System.getProperty(AbstractJdbcRegistryTest.class.getSimpleName()
+    protected static DatabaseType DEFAULT_DATABASE_TYPE = DatabaseType.H2;
+    protected static  DatabaseType DATABASE_TYPE = DatabaseType.valueOf(System.getProperty(AbstractJdbcRegistryTest.class.getSimpleName()
```

Comment:
looks like these changes can be reverted as well now, can't they?

Reply:
yes, i change it back to private final